configure.ac

```diff
@@ -219,24 +219,38 @@ AC_CHECK_LIB([gsl],[gsl_sf_bessel_Jn], [],
 # Check for HDF5 library
 AC_ARG_WITH(hdf5, [AC_HELP_STRING([--without-hdf5],[do not allow HDF5 output])], ok=$withval, ok=yes)
 if test "$ok" = "yes"; then
```

Comment:
I wouldn't recommend changing the configure.ac script at all in this PR if possible.  For example, if you want to use HDF5 from pkgconfig to find HDF5, you can just configure with
```sh
./configure LDFLAGS=`pkg-config --libs-only-L hdf5` CPPFLAGS=`pkg-config --cflags-only-I hdf5`
```
or something like that.

Reply:
We can update the configure script to use `pkg-config` in a separate PR if we want.